### PR TITLE
aligning headers

### PR DESCRIPTION
**Pull request by egfefey**
Aligned the headers

### CODE REVIEW

File: include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h [hunk 1]
```diff
@@ -14,6 +14,8 @@
 #include <memory>
 #include <ostream>
 #include <vector>
+#include <exception>
```
Comment (AnuyaWelling2801) [inline include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h:17]: Why does this file have #include <exception> and the rest don't ?

File: test/support/concurrent_queue.h
Comment (AnuyaWelling2801) [inline test/support/concurrent_queue.h:1]: This files should be removed too

File: include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h (hunk 1 above)
Comment (egfefey) [reply include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h:17]: Do you mean #include <exception>?

Comment (egfefey) [reply include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h:17]: Do you mean #include exception? Only dynamic load and auto tune use it.

File: include/oneapi/dpl/internal/dynamic_selection_impl/auto_tune_policy.h [hunk 2]
```diff
@@ -16,7 +16,11 @@
 #include <memory>
 #include <mutex>
 #include <utility>
+#include <chrono>
```
Comment (vossmjp) [inline include/oneapi/dpl/internal/dynamic_selection_impl/auto_tune_policy.h:18]: seems like atomic might not be needed.

File: include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h (hunk 1 above)
Comment (vossmjp) [inline include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h:17]: ostream should be removed. And I think mutex and limits are missing.

File: test/support/concurrent_queue.h [hunk 3]
```diff
@@ -14,6 +14,7 @@
 #include <mutex>
 #include <deque>
 #include <condition_variable>
+#include <list>
```
Comment (vossmjp) [inline test/support/concurrent_queue.h:17]: I think this whole header is no longer needed. And should be removed, along its include in the inline_backend

**Review by vossmjp — COMMENTED** [review]
Still a few missing / extra includes I think.

File: include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h (hunk 1 above)
Comment (egfefey) [reply include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h:18]: Done

File: test/support/concurrent_queue.h
Comment (egfefey) [reply test/support/concurrent_queue.h:1]: Done

File: include/oneapi/dpl/internal/dynamic_selection_impl/auto_tune_policy.h (hunk 2 above)
Comment (egfefey) [reply include/oneapi/dpl/internal/dynamic_selection_impl/auto_tune_policy.h:18]: Done

**Review by vossmjp — APPROVED** [review]
Looks good.

**Review by AnuyaWelling2801 — APPROVED** [review]

**Pull request merged**